sdk/include/asm/asm.inc

```diff
@@ -320,12 +322,23 @@ ENDM
 #define REPEAT .rept
 #define ENDR .endr
 
+#define lXdtPrefix
+
 .macro ljmp segment, offset
-    jmp far ptr \segment:\offset
+    .byte 0xEA
```

Comment:
What's wrong with `jmp far ptr \segment:\offset`?

Reply:
far jump is not supported in 64 bits compiler!

Reply:
> far jump is not supported in 64 bits compiler!

This may be worth adding as a comment in the code (and "compiler" -> "assembler")